touchforms/formplayer/static/formplayer/script/webformsession.js

```diff
@@ -143,37 +142,18 @@ WebFormSession.prototype.serverRequest = function (requestParams, callback, bloc
 
     this.numPendingRequests++;
     this.onLoading();
-
-    if (self.formplayerEnabled){
```

Comment:
actually this entire file is obsolete, but we shouldn't delete this in the HQ directory